packages/extension-ui/src/Popup/ImportQr.tsx

```diff
@@ -50,8 +52,12 @@ export default function ImportQr(): React.ReactElement {
           .then(() => onAction('/'))
           .catch((error: Error) => console.error(error));
       }
+
+      if (name && password) {
+        show(t('Account created successfully!'), 'success');
```

Comment:
account creation seems to be async, shouldn't we wait for possible errors?

Reply:
The logic was already implemented this way - the taost would should even before executing the password creation functions (I'll move it above to match it 100%). Besides, the async functions used here resolve right after sending the message over the communication channel - not after the account is created, so even if we waited, the toast would still not represent the real success. So since proper fixing is not possible, I think I'll leave it as it was.